app/models/pageflow/published_entry.rb

```diff
@@ -25,6 +25,7 @@ class PublishedEntry
              :author, :publisher, :keywords,
              :theme,
              :password_protected?,
+             :created_at, :updated_at,
```

Comment:
Indeed. If `touch: true` is ever removed, or it stops working, it won't invalidate the cache then. I guess it's a fair tradeoff, but if you'd prefer I can write two methods that are more bullet proof than a simple delegation. The methods could take a published revision, or fallback to the entry when there is none. It's probably the best solution, but more LOC.